stand-ups/readme.md

```diff
@@ -1,18 +1,18 @@
 # Stand-ups
 
-The stand-up is a ceremony that is held each day of the sprint. In this ceremony, each contributor will answer three simple questions. This will repeat until each contributor has answered the three following questions.
+The stand-up is a time-box ceremony that is held each day of the sprint. In this ceremony, each contributor in the Development Team will answer three simple questions. This will repeat until each contributor has answered the three following questions.
 
-1. What did you work on yesterday?
-2. What are you working on today?
-3. Do you have any impediments/blockers? (defer discussion / resolution to "the parking lot", described below)
+> 1. What did you work on yesterday that contributes to meet the sprint goal?
+> 2. What are you working on today that will contribute to meet the sprint goal?
+> 3. Do you have any impediments/blockers? (defer discussion / resolution to "the parking lot", described below)
 
 After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.
 
 The term parking lot refers to a bucket of comments, concerns, or questions that will be discussed and/or addressed at a later point with potentially fewer contributors. This is part of a strategy to avoid letting the discussion in a meeting shift to a subject that is not aligned with the meeting goals and/or decisions.
 
 ## Goals
 

```

Comment:
Identify and synchronize as a team
Coordinate efforts across the team to progress on goals and solve problems
Have a shared understanding of goals, problems, and progress
Reduce additional meetings

Reply:
Good points. before updating I will raise a question to the Agile team to gather a consensus and better understand on what need to be covered on standups to avoid that we end-up doing a status meeting that is not the purpose.

Reply:
It will be covered in the next update.